### ISSUE TITLE
The Default template is not accessible enough for Screen Reader users

### ISSUE THREAD

**Issue by hacdias** [filebrowser/filebrowser#131]
_From @Flameborn on April 4, 2017 10:17_

Currently, Caddy-FileManager's default template does not provide enough information to screen readers.
For example, in the file list, folders or files are not marked up properly as a link or a clickable element. Items are presented as DIV elements. Without proper ARIA markup, screen readers see this as standard text and are unable to click on any item to navigate to the selected folder, or download a file. Keyboard users are also unable to focus these items via the tab key.

The easiest solution is to use standard links, or if this is not possible due to styling, ARIA roles. See this article: https://www.marcozehe.de/2013/04/24/easy-aria-tip-6-making-clickables-accessible/

Screen reader support can be tested via VoiceOver under MAcOS and iOS, (cmd+f5, settings/general/accessibility/VoiceOver) or with the open source NVDA https://www.nvaccess.org under Windows.

_Copied from original issue: hacdias/filemanager-old#80_

**Comment by hacdias** [comment]
Hello @Flameborn! You're right. Unfortunately, I can't to this right now because I'll be some time away from home. But if you have got some spare time, you could make a Pull Request if you want 😄

**Comment by hacdias** [comment]
Just reviewed the whole HTML code and added the roles and ARIA markup wherever I thought it was needed. I'll close this. If you find something that should be improved, just file an issue or make a PR 😄

**Comment by Flameborn** [comment]
Thank you very much!
